src/solvers/dgsem_tree/dg_2d.jl

```diff
@@ -78,6 +78,27 @@ function create_cache(mesh::Union{TreeMesh{2}, StructuredMesh{2}, UnstructuredMe
   return (; fstar1_L_threaded, fstar1_R_threaded, fstar2_L_threaded, fstar2_R_threaded)
 end
 
+
+function create_cache(mesh::TreeMesh{2}, equations,
+                      volume_integral::VolumeIntegralShockCapturingSubcell, dg::DG, uEltype)
+  cache = create_cache(mesh, equations,
+                       VolumeIntegralPureLGLFiniteVolume(volume_integral.volume_flux_fv),
+                       dg, uEltype)
+
+  A3dp1_x = Array{uEltype, 3}
+  A3dp1_y = Array{uEltype, 3}
+  A3d = Array{uEltype, 3}
+
+  fhat1_threaded = A3dp1_x[A3dp1_x(undef, nvariables(equations), nnodes(dg)+1, nnodes(dg)) for _ in 1:Threads.nthreads()]
+  fhat2_threaded = A3dp1_y[A3dp1_y(undef, nvariables(equations), nnodes(dg), nnodes(dg)+1) for _ in 1:Threads.nthreads()]
+  flux_temp_threaded = A3d[A3d(undef, nvariables(equations), nnodes(dg), nnodes(dg)) for _ in 1:Threads.nthreads()]
+
+  ContainerAntidiffusiveFlux2D = Trixi.ContainerAntidiffusiveFlux2D{uEltype}(0, nvariables(equations), nnodes(dg))
```

Comment:
```suggestion
  container_antidiffusive_flux = Trixi.ContainerAntidiffusiveFlux2D{uEltype}(0, nvariables(equations), nnodes(dg))
```
Please use `snake_case` for variables